.github/workflows/ci-build.yml

```diff
@@ -56,3 +56,35 @@ jobs:
       uses: docker/build-push-action@v2
       with:
         file: ./facilitator/Dockerfile
+  
+  build-deploy-tool:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Setup go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.15
+    
+    - name: Get dependencies
+      run: go get -v -t -d ./...
+      working_directory: deploy-tool
+    
+    - name: Get linter
+      run: go install golang.org/x/lint/golint
```

Comment:
Do we run this linter after installing it? Is that what `go vet` does?

Reply:
...I don't know how I forgot this.